.changelog/unreleased/features/1606-multisignature-draft.md

```diff
@@ -0,0 +1,2 @@
+- Introduce multisignature account and transaction format.
```

Comment:
it would be nice to write a little bit more about this and/or link to docs for anyone who'll want to try it in the new release

Reply:
yes pls!